modules/widgets/google-translate.js

```diff
@@ -0,0 +1,16 @@
+function googleTranslateElementInit() {
```

Comment:
Could you move this file to its own subdirectory, like it's done for other widgets? `modules/widgets/google-translate/google-translate.js` would work.